<launch>
  <!-- optris drivers
  <node name="optris_imager_node" pkg="optris_drivers" type="optris_imager_node" output="screen">
    <param name="xmlConfig" type="string" value="$(find physiological_monitoring)/launch/15110057.xml"/>
  </node>
  -->
  
  <!-- rosbag play -->
  <arg name="bag"/>
  <param name="use_sim_time" value="true"/>
  <node pkg="rosbag" type="play" name="rosbag_play" args="--queue=1000 --clock $(arg bag)" required="true"/>
  
  <!-- ENRICHME Physiological Monitoring -->
  <param name="data_visualization" value="true"/>
  <node pkg="physiological_monitoring" type="physiological_monitoring" name="physiological_monitoring" output="screen" respawn="false"/>
  
  <!-- rviz -->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find physiological_monitoring)/launch/physiological_monitoring.rviz"/>
  
</launch>
